### PR TITLE
Ivysaur 8

### PR DESCRIPTION
**Pull request by Jianriu**

### CODE REVIEW

**Comment by camfiander** [comment]
```
PS C:\Users\camfi\git\on-call-coordinator\server> npx prisma migrate reset
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": MySQL database "oncall" at "localhost:42333"

√ Are you sure you want to reset your database? All data will be lost. ... yes

Applying migration `20221121184656_create_school_success`
Error: P3018

A migration failed to apply. New migrations cannot be applied before the error is recovered from. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

Migration name: 20221121184656_create_school_success

Database error code: 1146

Database error:
Table 'oncall.Teacher' doesn't exist

Please check the query number 1 from the migration file.

```
prisma machine broke

**Comment by camfiander** [comment]
I've updated the prisma migrations so they don't create errors.  For future DB changes just run `npx prisma migrate dev` without deleting the `migrations` folder, deleting `migrations` is more like a last resort when migrations are failing in your *local* env

File: client/src/page/viewMasterSchedule.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    }
+    
+    useEffect(() => {
+        getSchedule();
+    },[check])
+    useEffect(() => {
+        setRows(filterSchedule(data));
+    },[data])
+    return(
+        <div style={{ "& .header": {backgroundColor: "#a1f0ee"},
```
Comment (camfiander) [inline client/src/page/viewMasterSchedule.jsx:54]: backgroundColor doesn't seem to cover the whole width of the datagrid here.. are you able to put this in its own css file so it's easier to tweak later? Just thinking if we mess with colors or other styling stuff in the last couple days these might be hard to find

**Review by camfiander — CHANGES_REQUESTED** [review]
This is ready to merge once `specialCode` gets displayed somehow in the "Course Title"

File: client/src/page/viewMasterSchedule.jsx (hunk 1 above)
Comment (Jianriu) [reply client/src/page/viewMasterSchedule.jsx]: I tried but I could not get it to work. I will just notify whoever does tweak it to look here.

**Comment by Jianriu** [comment]
Should now show the specialCode as a title if class is null

**Review by camfiander — APPROVED** [review]

**Pull request merged**